src/Alert/Alert.levers.js

```diff
@@ -0,0 +1,16 @@
+const classnames = require('classnames')
+
+/**
+ * Default Types for the type prop.
+ */
+export const alertTypes = ['info', 'warning', 'error', 'success']
+
+/**
+ * Styles for light text
+ */
+export const lightText = classnames('su-text-white hover:su-link-no-underline')
```

Comment:
These probably need focus: classes here but I see that you're adding some stuff to decanter-7 so can wait till those are merged into v7 branch.